tools/arxiv/tests/run_large_scale_test.sh

```
@@ -0,0 +1,192 @@
+echo -e "\n${GREEN}Step 2: Setting up test environment${NC}"
+
+# Create staging directory if needed
+STAGING_DIR="/dev/shm/large_scale_staging"
+if [ ! -d "$STAGING_DIR" ]; then
+    echo "Creating staging directory: $STAGING_DIR"
+    mkdir -p "$STAGING_DIR"
+fi
+
+# Clear any previous test data
+echo "Clearing previous staging data..."
+rm -rf "$STAGING_DIR"/*
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)